qjs.c

```diff
@@ -403,7 +403,7 @@ int main(int argc, char **argv)
                     opt_arg = argv[optind++];
                 }
                 // TODO(chqrlie): accept kmg suffixes
-                memory_limit = (size_t)strtod(opt_arg, NULL);
+                memory_limit = (int)atoi(opt_arg);
                 break;
```

Comment:
This looks wrong to me: `memory_limit` should be a `size_t` to handle limits larger than 2GB.  I suggest using `strtoull(opt_arg, NULL, 0)`.  Handing suffixes for kilo-, mega- and gigabytes would be nice.
What is the unit for similar settings in `node` ?

Reply:
Another remark: `node` has similar options but the limits are expressed in kilobytes.  To reduce confusion and make them more user friendly, we should use kilobytes as a default and accept a suffix for other units.

Reply:
Got it. I'll try and get to this over the weekend.